watertap/core/zero_order_costing.py

```diff
@@ -1255,50 +1256,97 @@ def cost_fixed_bed(blk, number_of_parallel_units=1):
             blk.unit_model.cationic_polymer_demand[t0], "cationic_polymer"
         )
 
-    def cost_gac(blk):
+    def cost_gac(blk, number_of_parallel_units=5, gac_bulk_density=500):
         """
-        General method for costing granular activated carbon processes. Capital
-        cost is based on the inlet flow rate of liquid and the empty bed
-        contacting time.
-        This method also registers electricity and activated carbon consumption
-        as costed flows.
+        Adapted from core GAC costing model initially released in v0.6.0
+
+        3 equation capital cost estimation for GAC systems with: (i), contactor/pressure vessel cost by polynomial
+        as a function of individual contactor volume; (ii), initial charge of GAC adsorbent cost by exponential as a
+        function of required mass of GAC adsorbent; and (iii), other process costs (vessels, pipes, instrumentation, and
+        controls) calculated by power law as a function of total contactor(s) volume. Operating costs calculated as the
+        required makeup and regeneration of GAC adsorbent. Energy for backwash and booster pumps considered negligible
+        compared to regeneration costs
+        Args:
+            number_of_parallel_units (int, optional) - cost this unit as
+                        number_of_parallel_units parallel units in operation (default: 5)
+            gac_bulk_density (optional) - estimated bulk density
+                        (mass of gac [kg]/ volume of bed [m^3]) (default: 500)
```

Comment:
We'd better not hardcode these two variables, you can add `number_of_parallel_units` and `gac_bulk_density` as two separate parameters in the `gac.yaml` and the reference values could be saved in the yaml file. Then register `number_of_parallel_units` and `gac_bulk_density` as variables in the `gac_zo.py`

Reply:
`number_of_parallel_units` is utilized in this hardcoded method in all other ZO costing models, so I am hesitant on moving this. I think moving the `gac_bulk_density` makes sense and can move forward with this. What are your thoughts?

Reply:
I think we should eventually consider moving `number_of_parallel_units` to the yaml files, but we should do so globally and in a separate PR.

@luohezhiming would you mind opening an issue for this?

Reply:
Let's move `gac_bulk_density` to `gac.yml` in this PR.

Reply:
Sure. I am out of office right now, but I believe we need to consider a standard method for handling multiple units